packages/block-editor/src/components/block-list/index.js

```diff
@@ -80,8 +80,8 @@ function BlockList(
 
 	return (
 		<Container
-			{ ...__experimentalPassedProps }
 			ref={ ref }
+			{ ...__experimentalPassedProps }
```

Comment:
I just copied this change from #25515 for the sake of testing, since without it, the block doesn't work.